content/en/docs/tasks/administer-cluster/running-cloud-controller.md

```diff
@@ -46,17 +46,15 @@ integration, it should not be too different from the requirements when running
 
 Successfully running cloud-controller-manager requires some changes to your cluster configuration.
 
-* `kube-apiserver` and `kube-controller-manager` MUST NOT specify the `--cloud-provider`
-  flag. This ensures that it does not run any cloud specific loops that would be run by
-  cloud controller manager. In the future, this flag will be deprecated and removed.
-* `kubelet` must run with `--cloud-provider=external`. This is to ensure that the
-  kubelet is aware that it must be initialized by the cloud controller manager
-  before it is scheduled any work.
+* `kubelet`, `kube-apiserver` and `kube-controller-manager` must be set according to the
+user's usage of external CCM. If the user has an external CCM (not the internal cloud
+controller loops in the Kubernetes Controller Manager), then `--cloud-provider=external`
+must be specified. Otherwise, it should not be specified.
```

Comment:
Indent these lines by two spaces would be desirable.

Reply:
@tengqm 
Updated, Thank you.